src/main/java/services/petshop/PetShopService.java

```diff
@@ -1,4 +1,23 @@
 package services.petshop;
 
+import utils.constants.TestConstants;
+
 public class PetShopService {
+
+    public String baseUri;
+
+    public static final String POST_PET = "/pet";
```

Comment:
Since is the same endpoint but the operations are different, I would let only one endpoint. Suggestion: PET_URI